therapy/etl/ncit.py

```diff
@@ -0,0 +1,151 @@
+            )
+            db.add(other_id)
+        if leaf.P210:
+            other_id = OtherIdentifier(
+                concept_id=concept_id,
+                other_id=f"{NamespacePrefix.CASREGISTRY.value}:{leaf.P210.first()}"  # noqa F501
+            )
+            db.add(other_id)
+        if leaf.P319:
+            other_id = OtherIdentifier(
+                concept_id=concept_id,
+                other_id=f"{NamespacePrefix.FDA.value}:{leaf.P319.first()}"
+            )
+            db.add(other_id)
+        if leaf.P320:
+            other_id = OtherIdentifier(
+                concept_id=concept_id,
+                other_id=f"{NamespacePrefix.ISO.value}:{leaf.P320.first()}"
+            )
+            db.add(other_id)
+
+        for a in aliases:
+            alias = Alias(alias=a, concept_id=concept_id)
+            db.add(alias)
+
+    def get_desc_nodes(self, node: ThingClass,
+                       uq_nodes: Set[ThingClass]) -> Set[ThingClass]:
+        """Create set of unique subclasses of node parameter.
+        Originally called on C1909, Pharmacologic Substance.
+        """
+        children = node.descendants()
+        if children:
+            for child_node in children:
+                if child_node is not node:
+                    uq_nodes.add(child_node)
+                    self.get_desc_nodes(child_node, uq_nodes)
+        return uq_nodes
+
+    def get_typed_nodes(self, uq_nodes: Set[ThingClass],
+                        ncit: owl.namespace.Ontology) -> Set[ThingClass]:
+        """Get all nodes with semantic_type Pharmacologic Substance"""
+        graph = owl.default_world.as_rdflib_graph()
+        query_str = '''SELECT ?x WHERE
+        {
+            ?x <http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#P106>
+            "Pharmacologic Substance"
+        }'''
+        typed_results = set(graph.query(query_str))
+
+        retired_query_str = '''SELECT ?x WHERE {
+            ?x <http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#P310>
+            "Retired_Concept"
+        }
+        '''
+
+        retired_results = set(graph.query(retired_query_str))
+
+        typed_results = typed_results - retired_results
+
+        def parse_uri(uri):
+            uri.toPython().split('#')[1]
+
+        for result in typed_results:
+            # parse result as URI and get ThingClass object back from NCIt
+            class_object = ncit[result[0].toPython().split('#')[1]]
+            uq_nodes.add(class_object)
+        return uq_nodes
+
+    def _add_meta(self, db: Session):
+        meta_object = Meta(src_name=SourceName.NCIT.value,
+                           data_license="CC BY 4.0",
+                           data_license_url="https://creativecommons.org/licenses/by/4.0/legalcode",  # noqa F401
+                           version=self._version,
+                           data_url="https://evs.nci.nih.gov/ftp1/NCI_Thesaurus/archive/20.09d_Release/Thesaurus_20.09d.OWL.zip",)  # noqa F401
```

Comment:
```suggestion
                           data_url="https://evs.nci.nih.gov/ftp1/NCI_Thesaurus/archive/20.09d_Release/",)  # noqa F401
```
In ChEMBL, we gave the index to the page that contains the db. We might want to do this for ncit?

Reply:
done!